book/src/ch03/gba_memory_mapping.md

```diff
@@ -1,19 +1,19 @@
-# GBA Memory
+# GBA Memory Mapping
 
 The [GBA Memory Map](http://problemkaputt.de/gbatek.htm#gbamemorymap) has
 several memory portions to it, each with their own little differences. Most of
 the memory has pre-determined use according to the hardware, but there is also
 space for games to use as a scratch pad in whatever way the game sees fit.
 
-The memory ranges listed here are _inclusive_, so they end with a lot of `F`s
-and `E`s.
+The memory ranges listed here are _inclusive_, so they end with a lot of F's
+and E's.
```

Comment:
This might be one of those US/UK things, but when you're pluralizing a singular letter the US convention is to put the apostrophe. See [rule 6 of this page](https://www.grammarbook.com/punctuation/apostro.asp).

Reply:
To my mind the rule in English for pluralising a word is to add an S. There's no apostrophe because there's nothing being elided. Thus, "Ps and Qs", not "P's and Q's" since the latter reads like a possessor.